modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -100,29 +100,16 @@ body .ddm-user-view-content {
 .lfr-ddm-form-container {
 	word-break: normal;
 
-	.btn {
-		opacity: unset;
-	}
+	.custom-control-input:disabled ~ .custom-control-label {
+		::before {
+			background-color: #0b5fff;
```

Comment:
Hey @igor-franca, as we talked before, you should use `$primary` variable here, as per [Clay UI Color](https://clayui.com/docs/css/color.html).